finetune/full.py

```diff
@@ -175,18 +172,20 @@ def train(
 
         t1 = time.perf_counter()
         total_lengths += input_ids.size(1)
+        flops_per_batch = measured_flops(meta_model, input_ids.shape)
```

Comment:
In the description, you have pointed out that this takes 0.4 seconds if I understand correctly. But that is quite significant, and I assume it blocks the GPU and therefore impacts the efficiency of training. Shouldn't we make this optional? The flops are interesting for benchmarking and comparing models, but the average user of our scripts is probably just interested in finetuning itself, right?

For example, we could just leave the line commented out, with a comment above it.